lib/fakeroot.py

```diff
@@ -186,13 +186,13 @@
      "cmds": ["dnf", "rpm", "yum"],
      "each": ["fakeroot"] },
 
-   # Debian notes:
+   # Debian/Ubuntu notes:
    #
-   # 1. By default in recent Debians, apt(8) runs as an unprivileged user.
-   #    This makes *all* apt operations fail in an unprivileged container
-   #    because it can't drop privileges. There are multiple ways to turn the
-   #    “sandbox” off. As far as I can tell, none are documented, but this one
-   #    at least appears in google searches a lot.
+   # 1. In recent Debians based distributions, by default, `apt(8) runs as an
+   #    unprivileged user. This makes *all* apt operations fail in an
+   #    unprivileged container because it can't drop privileges. There are
+   #    multiple ways to turn the “sandbox” off. As far as I can tell, none are
+   #    documented, but this one at least appears in google searches a lot.
```

Comment:
Copy-edit, including line wrapping:

```suggestion
   # 1. In recent Debian-based distributions apt(8) runs as an unprivileged
   #    user by default. This makes *all* apt operations fail in an
   #    unprivileged container because it can't drop privileges. There are
   #    multiple ways to turn the “sandbox” off. AFAICT, none are documented,
   #    but this one at least appears in Google searches a lot.
```